prt.py

```diff
@@ -303,6 +306,25 @@ def transcode_local():
         if output and is_debug:
             log.debug(output.strip('\n'))
 
+def get_available_remote_servers():
+    config = get_config()
+    servers = config["servers"]
+
+    for hostname, host in servers.items():
+        log.debug("Getting load for host '%s'" % hostname)
+        host["load"] = get_system_load_remote(hostname, host["port"], host["user"])
+
+        if not host["load"]:
+            # If no load is returned, then it is likely that the host
+            # is offline or unreachable
+            log.debug("Couldn't get load for host '%s'" % hostname)
+            del servers[hostname]
```

Comment:
This will work in python 2.7, but not python 3.x. In python 3.x `dict.items()` returns an iterator, whereas in python 2.7 it does not.